e-commerce_fake-store/src/components/login-card/index.js

```diff
@@ -0,0 +1,68 @@
+import "./index.css";
+import { Link, useHistory } from "react-router-dom";
+import React, { useState } from "react";
+
+const LoginCard = () => {
+  const history = useHistory();
+
+  const [login, setlogin] = useState({
+    username: "",
+    password: "",
+  });
+  var existingusers = JSON.parse(localStorage.getItem("alluserdata"));
+
+  const submitHandler = () => {
+    existingusers.filter((users) => {
```

Comment:
Consider code refactoring as follows:

```
existingUsers.filter(({ username, password }) => {
      if (
        username === login.username &&
       password === login.password
      ) {
        localStorage.setItem("isLoggedin", true);
        localStorage.setItem("isSignedup", true);
        history.push("/home");
      } else if (!username || !password) {
        alert("Enter Username and Password carefully");
      }
     else {
        /* handle the ELSE case as well here */
      }
    });
  };
```

Why .filter() is used here in the submit handler? Are we returning any filtered data here on submission? 
Re-check this and change it accordingly.

@shubhamshukla91